homeassistant/components/velbus/climate.py

```diff
@@ -73,11 +75,14 @@ def target_temperature(self):
 
     def set_temperature(self, **kwargs):
         """Set new target temperatures."""
-        temp = kwargs.get(ATTR_TEMPERATURE)
-        if temp is None:
-            return
-        self._module.set_temp(temp)
-        self.schedule_update_ha_state()
+        try:
```

Comment:
Please only wrap the lines that can cause the exception, with try... except. In this case that should be `self._module.set_temp(temp)`.